examples/multiple_windows/lib/app/main_window.dart

```diff
@@ -0,0 +1,255 @@
+            if (window.parent == null && !window.isMainWindow) {
+              childViews.add(
+                WindowControllerRender(
+                  controller: window.controller,
+                  key: window.key,
+                  windowSettings: settings,
+                  windowManagerModel: windowManagerModel,
+                  onDestroyed: () => windowManagerModel.remove(window.key),
+                  onError: () => windowManagerModel.remove(window.key),
+                ),
+              );
+            }
+          }
+
+          return ViewCollection(views: childViews);
+        },
+      ),
+      child: Scaffold(
+        appBar: AppBar(title: const Text('Multi Window Reference App')),
+        body: Row(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Expanded(
+              flex: 60,
+              child: SingleChildScrollView(
+                scrollDirection: Axis.vertical,
+                child: _WindowsTable(windowManagerModel: windowManagerModel),
+              ),
+            ),
+            Expanded(
+              flex: 40,
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.stretch,
+                children: [
+                  ListenableBuilder(
+                    listenable: windowManagerModel,
+                    builder: (BuildContext context, Widget? child) {
+                      return _WindowCreatorCard(
+                        selectedWindow: windowManagerModel.selected,
+                        windowManagerModel: windowManagerModel,
+                        windowSettings: settings,
+                      );
+                    },
+                  ),
+                ],
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _WindowsTable extends StatelessWidget {
+  const _WindowsTable({required this.windowManagerModel});
+
+  final WindowManagerModel windowManagerModel;
+
+  @override
+  Widget build(BuildContext context) {
+    return ListenableBuilder(
+      listenable: windowManagerModel,
+      builder: (BuildContext context, Widget? widget) {
+        return DataTable(
+          showBottomBorder: true,
+          onSelectAll: (selected) {
+            windowManagerModel.select(null);
+          },
+          columns: const [
+            DataColumn(
+              label: SizedBox(
+                width: 20,
+                child: Text('ID', style: TextStyle(fontSize: 16)),
+              ),
+            ),
+            DataColumn(
+              label: SizedBox(
+                width: 120,
+                child: Text('Type', style: TextStyle(fontSize: 16)),
+              ),
+            ),
+            DataColumn(
+              label: SizedBox(width: 20, child: Text('')),
+              numeric: true,
+            ),
+          ],
+          rows: <DataRow>[
+            for (KeyedWindow controller in windowManagerModel.windows)
+              DataRow(
```

Comment:
Nit: Consider making each of these DataRows its own separate private widget that takes `controller` as a parameter.